src/App.js

```diff
@@ -12,6 +12,17 @@ const PassingProsComponent = ({ prop1, prop2 }) => {
   );
 };
 
+const PassingProsComponent2 = ({ prop1, prop2 }) => {
+  //const { isBoolean, ...rest } = props;
+  //return <div {...rest}>{isBoolean ? "True" : "False"}</div>;
+  return (
+    <div>
+      {prop1}
+      {prop2}
+    </div>
+  );
```

Comment:
**Suggestion:** In the new component 'PassingProsComponent2', you are passing a prop named 'prop3' which is not being used in the component. Either use the prop in the component or remove it from the props being passed.
```suggestion
const PassingProsComponent2 = ({ prop1, prop2 }) => {
  return (
    <div>
      {prop1}
      {prop2}
    </div>
  );
};
...
<PassingProsComponent2 prop1="1" prop2="2">
  123
</PassingProsComponent2>
```